### ISSUE TITLE
VK_NVX_raytracing does not document structure of the instance buffer

### ISSUE THREAD

**Issue by kennyalive** [KhronosGroup/Vulkan-Docs#813]
vkCmdBuildAccelerationStructureNVX function has VkBuffer  instanceData parameter but it seems there is no description of instance buffer data layout in the specification of version 1.1.86.

**Comment by kayru** [comment]
While we wait for the official spec clarification, it's possible to use DXR spec to guess the instance data structure. It is likely to be this:

```
struct Instance
{
	float transform[12];
	uint32_t instanceID : 24;
	uint32_t instanceMask : 8;
	uint32_t instanceContributionToHitGroupIndex : 24;
	uint32_t flags : 8;
	uint64_t accelerationStructureHandle;
};
```
Acceleration structure handle should be acquired via `vkGetAccelerationStructureHandleNVX`.

**Comment by dgkoch** [comment]
what @kayru wrote is the correct layout -- it is the same as in DXR. 

We'll aim to correct this in a future documentation update.

**Comment by krOoze** [comment]
@dgkoch BTW `VkAccelerationStructureTypeNVX` also missing. And `VkGeometryInstanceFlagBitsNVX` marked as "TBD" (part of the Instance Buffer??).

**Comment by dgkoch** [comment]
VK_NVX_raytracing has now been replaced with VK_NV_ray_tracing which is now published in the 1.1.91 spec (https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#VK_NV_ray_tracing ).  The documentation has also significantly improved and the issues reported here should all be addressed.
